PowerShellEditorServices.build.ps1

```diff
@@ -5,19 +5,48 @@
 
 param(
     [ValidateSet("Debug", "Release")]
-    [string]$Configuration = "Debug"
+    [string]$Configuration = "Debug",
+
+    [string]$PsesSubmodulePath = "$PSScriptRoot/module",
+
+    [string]$ModulesJsonPath = "$PSScriptRoot/modules.json",
+
+    [string]$DefaultModuleRepository = "PSGallery"
 )
 
 #Requires -Modules @{ModuleName="InvokeBuild";ModuleVersion="3.2.1"}
 
 $script:IsCIBuild = $env:APPVEYOR -ne $null
 $script:IsUnix = $PSVersionTable.PSEdition -and $PSVersionTable.PSEdition -eq "Core" -and !$IsWindows
 $script:TargetFrameworksParam = "/p:TargetFrameworks=\`"$(if (!$script:IsUnix) { "net451;" })netstandard1.6\`""
+$script:SaveModuleSupportsAllowPrerelease = (Get-Command Save-Module).Parameters.ContainsKey("AllowPrerelease")
 
 if ($PSVersionTable.PSEdition -ne "Core") {
     Add-Type -Assembly System.IO.Compression.FileSystem
 }
 
+function Import-SubmoduleConfigurations
```

Comment:
Is this function needed?

I would think that in RestorePsesModules you could just do:

```powershell
    $modules = Get-Content -Raw "$PSScriptRoot/modules.json" | ConvertFrom-Json
    $modules.PSObject.Properties | ForEach-Object {
        $params = @{
            Name = $_.Name
            MinimumVersion = $_.Value.MinimumVersion
            MaximumVersion = $_.Value.MaximumVersion
            Path = "$PSScriptRoot/modules/"
        }
        if ($script:SaveModuleSupportsAllowPrerelease)        {
            $params += @{ AllowPrerelease = 
            $moduleInstallDetails.AllowPrerelease }
        }
        Save-Module @params
    }
```